application/src/main/resources/thingsboard.yml

```diff
@@ -107,6 +107,18 @@ coap:
   adaptor:  "${COAP_ADAPTOR_NAME:JsonCoapAdaptor}"
   timeout: "${COAP_TIMEOUT:10000}"
 
+#Quota parameters
+quota:
+  host:
+    limit: "${QUOTA_HOST_LIMIT:10000}"
```

Comment:
Please add one-two lines of comments for this parameters, because it is hard to guess from their names